contracts/governance/GraphGovernance.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+import "../upgrades/GraphUpgradeable.sol";
+
+import "./GraphGovernanceStorage.sol";
+
+/**
+ * @title Graph Governance Contract
+ * @notice Governance contract used to inscribe Graph Council and community votes.
+ */
+contract GraphGovernance is GraphGovernanceV1Storage, GraphUpgradeable, IGraphGovernance {
+    // -- Events --
+
+    /**
+     * @dev Emitted when `governor` calls createProposal()
+     */
+    event ProposalCreated(
+        bytes32 indexed proposalId,
+        bytes32 votes,
+        bytes32 metadata,
+        ProposalResolution resolution
+    );
+    /**
+     * @dev Emitted when `governor` calls updateProposal()
+     */
+    event ProposalUpdated(
+        bytes32 indexed proposalId,
+        bytes32 votes,
+        bytes32 metadata,
+        ProposalResolution resolution
+    );
+
+    /**
+     * @notice Initialize this contract.
+     */
+    function initialize(address _governor) public onlyImpl {
+        require(_governor != address(0), "governor !0");
```

Comment:
What this the intended error message? or was it "!governor"

Reply:
IIRC i was thinking `!governor` is more appropriate for when you are trying to access a function that only the governor can call. Maybe we can just name this error message to "Cannot set governor  = 0" or "governor != 0". 

Or , we can just go with `!governor` . What do you prefer?

Reply:
i went with `governor != 0`

Reply:
Makes sense